internal/util/option/option.go

```diff
@@ -27,13 +29,13 @@ func (o Option[T]) IsNone() bool {
 }
 
 func (o Option[T]) MustUnwrap() T {
-	if o.IsSome() {
-		return o.value.(T) //nolint:forcetypeassert // Type T is guaranteed
+	if v, isSome := o.Unwrap(); isSome {
+		return v
 	}
 
 	var t T
 
-	panic(fmt.Sprintf("MustUnwrap called on a None Option of type %T.", t))
+	panic(fmt.Sprintf("MustUnwrap called on a None Option[%T].", t))
```

Comment:
don't panic here

Reply:
But thats the point of this method. This is useful in tests and in cases where you know the option is guaranteed to be `Some` or in cases where having it `None` should cause a panic. If the programmer doesn't want to have a chance of the panic occuring then they can use `Unwrap()`.

Reply:
I have some concerns about the panic in this method, if this panic occurs during a crucial process, it could affect the overall service availability.

Perhaps, instead of panicking, could we consider returning an error in cases where the option is None?

What do you think about this approach? probably I didn't see full view of organization.

Reply:
I think I will remove it because its not currently used and the only place it was supposed to be used is in tests. But no function returns an Option. If something is "found/not found" the function would return `(T, bool)`. So you cant even use it in tests.

Reply:
The reason for returning a `T, bool` instead of an option is because if you want to get a value from an option you have to call Unwrap which gives you a `T` and a bool. So.. just return a T and a bool.

Reply:
let's remove it